app/routes.js

```diff
@@ -59,6 +60,10 @@ router.get('*/eligibility', (req, res, next) => {
     const data = req.session.data;
     // probably create an array of props and check the obj includes them all
     if (data['caller-full-name'] && data['caller-phone-number'] && data['deceased-national-insurance'] && data['deceased-full-name']) {
+      if (duplicateNinos.indexOf(data['deceased-national-insurance'].replace(/\s+/g, '')) !== -1) {
```

Comment:
```suggestion
      if (duplicateNinos.includes(data['deceased-national-insurance'].replace(/\s+/g, ''))) {
```
also may be worth assigning the stripped nino to a var and using that to clean up the if a bit?